packages/account/src/providers/scalars.ts

```diff
@@ -0,0 +1,50 @@
+import { type IDateTime, DateTime } from '@fuel-ts/utils';
+import { GraphQLScalarType, Kind } from 'graphql';
+
+export const UnixMillisecondScalar = new GraphQLScalarType<IDateTime, number>({
+  name: 'UnixMilliseconds',
+  description: 'DateTime custom scalar type',
+  serialize(value: unknown): number {
+    if (value instanceof DateTime) {
+      return value.toUnixMilliseconds();
+    }
+
+    throw new Error('Unable to serialize value');
+  },
+  parseValue(value: unknown): IDateTime {
+    if (typeof value === 'number') {
+      return DateTime.fromUnixMilliseconds(value);
+    }
+    throw new Error('Unable to parse value');
+  },
+  parseLiteral(ast): IDateTime {
+    if (ast.kind === Kind.INT) {
+      return DateTime.fromUnixMilliseconds(Number(ast.value));
+    }
+    throw new Error('Unable to parse literal');
+  },
+});
+
+export const Tai64Scalar = new GraphQLScalarType<IDateTime, string>({
+  name: 'Tai64',
+  description: 'Tai64 custom scalar type',
+  serialize(value: unknown): string {
+    if (value instanceof DateTime) {
+      return value.toTai64();
+    }
```

Comment:
Is this custom Scalar being used? How it communicates with `codegen` ?

I am asking this because after some investigation I didn't find a way to use custom scalars with `codegen` within a Node Graphql client application. 

Since we have an [issue](https://github.com/FuelLabs/fuels-ts/issues/1572) related to this, I am curious if this is possible or if we need to close this issue.

Reply:
Hey @Torres-ssf,

It is not being used (assumed that would come later), I just wanted to show that the `DateTime` implemented, does meet the requirements, such that, it can be used as a GraphQL Scalar.

There been a lot of talk about custom scalars for client-side based applications, however, can't find much that isn't Apollo specific. Potentially we could draw inspiration from a package written for Apollo [apollo-client-transformers](https://github.com/cult-of-coders/apollo-client-transformers/tree/master).

References #1572

Reply:
Decided to remove the scalars.
[6dd0716851556358d2dd0b6125642eec86d2c605](https://github.com/FuelLabs/fuels-ts/commit/6dd0716851556358d2dd0b6125642eec86d2c605)